include/stronk/unit_v2.h

```diff
@@ -70,26 +96,34 @@ using stronk_default_unit = unit<Tag,
 template<unit_like UnitT, typename UnderlyingT>
 using unit_value_t = typename UnitT::template value<UnderlyingT>;
 
+template<typename ScaleT, unit_like UnitT, typename UnderlyingT>
+using unit_scaled_value_t = typename UnitT::template scaled_t<ScaleT>::template value<UnderlyingT>;
+
 /**
  * @brief Lookup which specific type is the type for the given Dimensions.
  */
 template<::twig::dimensions_like DimensionsT>
 struct unit_lookup
 {
-    using unit_t = stronk_default_unit<DimensionsT>;
+    template<typename ScaleT>
+    using unit_t = stronk_default_unit<DimensionsT, ScaleT>;
```

Comment:
This does all the magic, right? Now I could do the `using MW = W::scaled_t< std::mega>` and use MW everywhere?

Reply:
Yup exactly!